src/alpaka/bin/main.cc

```diff
@@ -45,6 +46,9 @@ namespace {
 }  // namespace
 
 int main(int argc, char** argv) {
+  // Initialiase all supported devices
+  initialise();
```

Comment:
In principle this could initialize only the backends chosen by command line arguments, but I guess it makes little difference in practice.

Reply:
good point... I'll think if I can find a way to do that, while juggling the namespaces.